models/bg.model.js

```diff
@@ -146,4 +146,24 @@ exports.fetchUsers = () => {
         .then((users) => {
             return users.rows
         })
+}
+
+exports.selectCommentByID = (comment_id) => {
+    return db
+        .query(
+            `SELECT * FROM comments WHERE review_id = $1 ORDER BY created_at DESC`,
+            [comment_id])
+        .then((comment) => {
+            return comment.rows[0]
+        })
+}
+
+exports.removeCommentByID = (comment_id) => {
+    return db   
+        .query(
+            `DELETE FROM comments WHERE comment_id = $1`, [comment_id]
+        )
+        .then((comment) => {
+            return comment.rows[0]
```

Comment:
we don't need to return anything out of this as there's nothing to send back, the 204 status is enough proof that our delete request was successful